{{ cookiecutter.__package_name_kebab_case }}/.gitlab-ci.yml

```
@@ -1,103 +1,74 @@
 stages:
-  - prebuild
   - build
   - test
   - {% if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int %}deploy{% else %}publish{% endif %}
 
-Compute CI image hash:
-  stage: prebuild
-  script:
-    - export DOCKER_IMAGE_SHA="$(sha1sum Dockerfile poetry.lock pyproject.toml | sha1sum | cut -c 1-12)"
-    - echo "CI_IMAGE_SHA=$DOCKER_IMAGE_SHA" >> .env
-  artifacts:
-    reports:
-      dotenv: .env
+variables:
+  DOCKER_TLS_CERTDIR: "/certs"
+
+.parallel:
+  parallel:
+    matrix:
+      - PYTHON_VERSION: [{{ cookiecutter.python_version }}]
```

We need to add quote marks to avoid [this issue](https://gitlab.com/gitlab-org/gitlab/-/issues/349209):

```suggestion
      - PYTHON_VERSION: ["{{ cookiecutter.python_version }}"]
```